lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -820,6 +820,22 @@ def _apply_overrides(self) -> None:
                         """
                     )
                 )
+            case "slurmdbd":
+                _logger.debug("Creating slurmdbd service override.")
+                slurmdbd_service_override = Path(
+                    "/etc/systemd/system/slurmdbd.service.d/10-slurmdbd-exec-pre-pid.conf"
+                )
+                slurmdbd_service_override.parent.mkdir(exist_ok=True, parents=True)
+                slurmdbd_service_override.write_text(
+                    textwrap.dedent(
+                        """
+                        [Service]
+                        ExecPreStart=/bin/mkdir /var/run/slurmdbd
+                        ExecPreStart=/bin/chown slurm /var/run/slurmdbd
```

Comment:
```suggestion
                        PermissionsStartOnly=True
                        RuntimeDirectory=slurmdbd
                        RuntimeDirectoryMode=0755
                        
```
Possibly better to leverage systemd options?